src/views/viewer/SearchView/SearchResults/SearchResults.tsx

```diff
@@ -99,7 +97,7 @@ export const SearchResults: React.FC<SearchResultsProps> = React.memo(({ query }
             )}
           </FiltersWrapper>
         </PaddingWrapper>
-        <FiltersBar {...filtersBarLogic} categories={categories} />
+        <FiltersBar {...filtersBarLogic} activeFilters={['date', 'length', 'other', 'language']} />
```

Comment:
This one no longer supports categories filter?

Reply:
No, my bad!